src/plugin/keypress.ts

```diff
@@ -178,7 +172,17 @@ export default function (mind: MindElixirInstance) {
       // input
       return
     }
-    const keyHandler = key2func[e.keyCode]
+    const keyHandler = key2func[e.key]
     keyHandler && keyHandler(e)
   }
+
+  mind.map.onwheel = e => {
+    e.preventDefault()

```

Comment:
Maybe we should move the preventDefault() call inside the if condition?

Reply:
Sure, FYI, I only mimicked what's already in place [here for onkeydown](https://github.com/SSShooter/mind-elixir-core/blob/master/src/plugin/keypress.ts#L174).